dependency-check-suppressions.xml

```diff
@@ -47,13 +47,21 @@
 	</suppress>
 
 	<suppress until="2022-06-25">
-    		<notes><![CDATA[
-   			file name: spring-security-core-5.4.7.jar
-   		]]></notes>
-    		<packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-core@.*$</packageUrl>
-    		<vulnerabilityName>CWE-862: Missing Authorization</vulnerabilityName>
-  	</suppress>
+		<notes><![CDATA[
```

Comment:
These unwanted suppressions needs to be removed (I think you might have added it to keep the build green, but these were fixed last week)